pkg/roachpb/BUILD.bazel

```diff
@@ -263,6 +263,7 @@ genrule(
     name = "gen-batch-generated",
     outs = ["batch_generated-gen.go"],
     cmd = """
+     export COCKROACH_INTERNAL_DISABLE_METAMORPHIC_TESTING=true
```

Comment:
This one should become unnecessary when https://github.com/cockroachdb/cockroach/pull/76351 is merged as I think that breaks the dependency that pulls in metamorphic constants anyway.